targets/TARGET_STM/TARGET_STM32L0/device/stm32l0xx_hal_spi.c

```diff
@@ -1083,7 +1107,15 @@ HAL_StatusTypeDef HAL_SPI_Receive_IT(SPI_HandleTypeDef *hspi, uint8_t *pData, ui
     hspi->State        = HAL_SPI_STATE_BUSY_RX;
     hspi->ErrorCode    = HAL_SPI_ERROR_NONE;
 
-    hspi->RxISR        = &SPI_RxISR;
+    /* Set the function for IT treatment */
+    if(hspi->Init.DataSize > SPI_DATASIZE_8BIT )
+    {
+        hspi->RxISR = SPI_RxISR_16BIT;
+    }
+    else
+    {
+        hspi->RxISR = SPI_RxISR_8BIT;
```

Comment:
Indentation here is inconsistent with the rest of the function which uses 2 spaces